presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorDistributedQueries.java

```diff
@@ -145,4 +150,86 @@ public void testCreateBucketedTable()
 
         assertUpdate("DROP TABLE orders_bucketed");
     }
+
+    @Test
```

Comment:
Put this in `TestRaptorIntegrationSmokeTest`. I recently moved everything out of `TestRaptorDistributedQueries` and into the smoke test class.